src/hyperion/experiment_plans/robot_load_then_centre_plan.py

```diff
@@ -182,6 +184,8 @@ def robot_load():
 
         yield from bps.wait("robot_load")
 
+        yield from bps.abs_set(composite.thawer.thaw_for_time_s, params.thawing_time)
```

Comment:
Is there some point in this plan where we need to check that the thawer is off before continuing?

Reply:
I don't think so. I will double check with Neil

Reply:
In fact @neil-i03 do we need to wait at some point for the thawing to have stopped or we just fire and forget? It's ok to be thawing whilst collecting

Reply:
fire and forget. We can collect while thawing (and may have to).